openstack_controller/tests/test_unit_ironic.py

```diff
@@ -410,9 +424,9 @@ def test_ironic_nodes_pagination(
             if instance.get("ironic_microversion", None) != "1.80"
             else 'http://127.0.0.1:6385/baremetal/v1/nodes'
         )
-        assert args_list.count(baremetal_url) == api_call_count
+        assert args_list.count(baremetal_url) == expected_api_call_count
     if api_type == ApiType.SDK:
-        assert connection_baremetal.nodes.call_count == 1
+        assert connection_baremetal.nodes.call_count == expected_api_call_count
 
 
```

Comment:
I miss here the equivalent test `test_pagination_invalid_no_exception` you added for nova

Reply:
Added one!